dds/idl/value_reader_generator.cpp

```diff
@@ -103,6 +103,52 @@ namespace {
     }
   }
 
+#if OPENDDS_HAS_MAP
+  void map_helper(const std::string& expression, AST_Map* map,
+                       const std::string& idx, int level)
+  {
+    const bool use_cxx11 = be_global->language_mapping() == BE_GlobalData::LANGMAP_CXX11;
+    const std::string indent(level * 2, ' ');
+    const std::string kidx = "k" + idx;
+    be_global->impl_ <<
+      indent << "if (!value_reader.begin_map()) return false;\n" <<
+      indent << "for (" << (use_cxx11 ? "size_t " : "unsigned int ") << idx << " = 0; "
+        "value_reader.pairs_remaining(); ++" << idx << ") {\n";
+
+    std::string key_name = scoped(map->key_type()->name());
```

Comment:
```suggestion
    const std::string key_name = scoped(map->key_type()->name());
```
make anything that can be const const